app/status/healthcheck.py

```diff
@@ -19,10 +19,10 @@ def show_status():
     else:
         return jsonify(
             status="ok",  # This should be considered part of the public API
-            travis_commit=version.__travis_commit__,
-            travis_build_number=version.__travis_job_number__,
-            build_time=version.__time__,
-            db_version=get_db_version()), 200
```

Comment:
We are using the db_version in this file to see if we need to test the migrations when deploying api https://github.com/alphagov/notifications-api/blob/0340927a6e0eb7a5e20bce6e569744f4f1895502/scripts/check_if_new_migration.py

Reply:
Ah shoot... We might be able to use a different strategy for that. Good catch